prismic-model/json/background-textures.json

```diff
@@ -0,0 +1,13 @@
+{
+  "BackgroundTexture" : {
+    "backgroundTexture" : {
+      "type" : "Image"
+    },
+    "backgroundName" : {
+      "type" : "Text",
+      "config" : {
+        "label" : "Background name"
+      }
+    }
+  }
+}
```

Comment:
Are we expecting a certain ratio?
Also - could we keep the property name conventional - i.e. `name` and `image`?

Reply:
The one we have so far is square – can't be sure that this will be true for all of them. I suppose it would make sense if they all did. Will check with Gareth O.

What would be the implications if so?

Reply:
This is how we force ratios else where:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/prismic-model/js/parts/promo.js#L24-L36